core/src/main/java/com/linecorp/armeria/common/util/Sampler.java

```diff
@@ -43,35 +43,8 @@
  * <p>Forked from brave-core 5.6.3 at 3be55b5cccf881104bdd80c93e97d2575b83952d
  */
 // abstract for factory-method support on Java language level 7
```

Comment:
So this is not true anymore. 😉

Reply:
Thank you for review. fiexed in 6459312.